versions/1/data_stream/fields/fields.spec.yml

```diff
@@ -52,20 +52,33 @@ spec:
         - wildcard
         - version
         - unsigned_long
+
       description:
         description: Short description of field
         type: string
+
       value:
         description: The value to associate with a constant_keyword field.
         type: string
+
       metric_type:
-        description: Metric type
+        description: >
+          The metric type of a numeric field. This is attached to the
+          field as metadata (via `meta`). A gauge is a single-value measurement
```

Comment:
`metric_type` is added as a `time_series_metric` mapping in a recent [change](https://github.com/elastic/kibana/pull/126322), should this be mentioned here?

Reply:
Thanks for letting me know. I've update the docs for this. I also updated the `dimension` parameter to mention that it uses `time_series_dimension`.